src/profile/v100/entity/persona/persona.rs

```diff
@@ -252,6 +258,30 @@ impl Persona {
         )
     }
 
+    pub fn sample_mainnet_turing() -> Self {
```

Comment:
new sample values with `EntityFlags` with `DeletedByUser`, ensured other sample values do not have this flag, so that AuthorizedPersonaSimple references non hidden accounts and personas...